microsoft-sentinel/info.json

```diff
@@ -0,0 +1,3125 @@
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "WorkspaceSubscriptionId"
+        },
+        {
+          "title": "Workspace Resource Group",
+          "description": "Azure active directory resource group from which you want to delete the indicator in Microsoft Sentinel.",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "WorkspaceResourceGroup"
+        },
+        {
+          "title": "Workspace Name",
+          "description": "Name of the workspace from which you want to delete the indicator in Microsoft Sentinel.",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "WorkspaceName"
+        }
+      ],
+      "output_schema": {
+        "result": ""
+      },
+      "enabled": true
+    },
+    {
+      "title": "Get Incident List",
+      "description": "Retrieves all incidents from Microsoft Sentinel based on the workspace subscription ID, workspace resource group, workspace name, and other input parameters that you have specified.",
+      "operation": "get_incident_list",
+      "category": "investigation",
+      "annotation": "get_incident_list",
+      "parameters": [
+        {
+          "title": "Workspace Subscription ID",
+          "description": "Azure active directory subscription ID from which you want to retrieve the incident from Microsoft Sentinel.",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "WorkspaceSubscriptionId"
+        },
+        {
+          "title": "Workspace Resource Group",
+          "description": "Azure active directory resource group from which you want to retrieve the incident from Microsoft Sentinel.",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "WorkspaceResourceGroup"
+        },
+        {
+          "title": "Workspace Name",
+          "description": "Name of the workspace from which you want to retrieve the incident from Microsoft Sentinel.",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "WorkspaceName"
+        },
+        {
+          "title": "Created DateTime",
+          "description": "(Optional) Created Datetime based on which you want to retrieve incidents from Microsoft Sentinel.",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "datetime",
+          "name": "created_datetime",
+          "tooltip": "(Optional) Created Datetime based on which you want to retrieve incidents from Microsoft Sentinel."
+        },
+        {
+          "title": "Search Query",
```

Comment:
add filter param like Status, Severity , category instead of single search query.

Reply:
Need to discuss for this parameter.